config/node/config.toml

```diff
@@ -150,6 +151,9 @@ CleanupLockedProofsInterval = "2m"
 GeneratingProofCleanupThreshold = "10m"
 UpgradeEtrogBatchNumber = 0
 BatchProofL1BlockConfirmations = 2
+SettlementBackend = "l1"
```

Comment:
what does this do? what's the default value for this?

Reply:
whether to use the agg layer or not, [the value is agglayer or l1].(https://github.com/0xPolygon/cdk-validium-node/blob/v0.6.4%2Bcdk/aggregator/config.go#L11-L20)

Reply:
got it. yeah i also found it https://github.com/0xPolygon/cdk-validium-node/blob/v0.6.4%2Bcdk/config/default.go#L178